### PR TITLE
Update Broken Link at Meshery Docs Nginx-SM Page

### LINKED ISSUE #2726: [Docs] Broken link on Nginx Service Mesh page
**Current State:**
The [link](https://docs.meshery.io/service-meshes/adapters/nginx-sm) to check progress on the Nginx Service Mesh adapter currently returns a 404. 

image

**Desired State:**
Add an appropriate link to the page.

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

**Description**

This PR fixes #2726 

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_adapters/nginx-sm.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ permalink: service-meshes/adapters/nginx-sm
 ---
 {% include adapter-status.html %}
 
-The {{ page.name }} is currently under construction ({{ page.project_status }} state). Want to contribute? Check our [progress](page.github_link).
-
+The {{ page.name }} is currently under construction ({{ page.project_status }} state). Want to contribute? Check our [progress](https://github.com/layer5io/meshery-nginx-sm).
```
Comment (Nikhil-Ladha) [inline docs/_adapters/nginx-sm.md:15]: Actually, the link is correct but the way it was rendered was incorrect. Check the code at the bottom of this file, where the same link is used and update here accordingly.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: docs/_adapters/nginx-sm.md (hunk 1 above)
Comment (leecalcote) [inline docs/_adapters/nginx-sm.md:15]: Let's use `{{ page.github_link }}` instead.

**Review by leecalcote — DISMISSED** [review]

File: docs/_adapters/nginx-sm.md (hunk 1 above)
Comment (leecalcote) [reply docs/_adapters/nginx-sm.md:15]: Ditto as to what @Nikhil-Ladha said.

Comment (Nikhil-Ladha) [reply docs/_adapters/nginx-sm.md:15]: Haa! i was 6 seconds late😂

Comment (leecalcote) [reply docs/_adapters/nginx-sm.md:15]: Minus the round-trip latency from GitHub's server (wherever that is), and we're tied. 😄

Comment (adithyaakrishna) [reply docs/_adapters/nginx-sm.md]: @leecalcote I tried that but if that is used I wasn't sure if the link would be attached to this text
As the URLs in Markdown required `[ ]( ) --> This Format`
image

Comment (adithyaakrishna) [reply docs/_adapters/nginx-sm.md]: But I've fixed that in the latest comment :)

**Comment by adithyaakrishna** [comment]
@Nikhil-Ladha Can you review this again please? 🥺

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**